README.md

```diff
@@ -7,24 +7,28 @@
   <p align="center">Open-source, end-to-end encrypted tool to manage secrets and configs across your team and infrastructure.</p>
 </p>
 
+# Deprecated!
+
+This is now considered a legacy SDK, as we have released a new SDK that will be receiving all future updates. [You can find it here](https://www.npmjs.com/package/@infisical/sdk).
+
 # Table of Contents
 
-- [Links](#links)
-- [Basic Usage](#basic-usage)
-- [Secrets](#working-with-secrets)
-  - [Get Secrets](#get-secrets)
-  - [Get Secret](#get-secret)
-  - [Create Secret](#create-secret)
-  - [Update Secret](#update-secret)
-  - [Delete Secret](#delete-secret)
-- [Cryptography](#cryptography)
-  - [Create Symmetric Key](#create-symmetric-key)
-  - [Encrypt Symmetric](#encrypt-symmetric)
-  - [Decrypt Symmetric](#decrypt-symmetric)
+-   [Links](#links)
```

Comment:
It looks like your editor added lots of extra spaces. Is it possible you can remove these so the pr reflects only the changes you made?

Reply:
Yep. I spoke to Tony about this prior to the PR, and we didn't seem to have a standard for formatting, which has led to a lot of extra spaces

Reply:
Yep. I spoke to Tony about this prior to the PR, and we didn't seem to have a standard for formatting, which has led to a lot of extra spaces